crab-pre.spec

```diff
@@ -2,11 +2,11 @@
 #For new crabclient_version, set the version_suffix to 00
 #For any other change, increment version_suffix
 ##########################################
-%define version_suffix 02
-%define crabclient_version v3.210323
+%define version_suffix 00
+%define crabclient_version v3.210825
 ### RPM cms crab-pre %{crabclient_version}.%{version_suffix}
 %define wmcore_version     1.3.6.crab6
-%define crabserver_version v3.210318
+%define crabserver_version v3.2106p11
```

Comment:
@belforte crabserver version should be v3.210611p1

Reply:
bad copy/paste. sorry